src/firmware/app/control/control.c

```diff
@@ -51,7 +51,7 @@ float app_control_getMaximalTorqueScaling(const ForceWheel_t* force_wheels[4],
         }
     }
 
-    float emf_ratio_min = battery_voltage / max_effective_motor_voltage;
+    float emf_ratio_min = battery_voltage / (max_effective_motor_voltage + (float)1.0e-9);
```

Comment:
Motion is now so accurate it exposed some divide-by-zero bugs

Reply:
Is this constant documented anywhere? Like for example, why does this become the defined behaviour for division by 0 instead of something else?

Reply:
No the constant is arbitrary and small, it's just my preferred way of dealing with potential division by zero since it doesn't require a bunch of `if-else` conditions.